python/cdp/evm_server_account.py

```diff
@@ -711,6 +711,33 @@ async def wait_for_fund_operation_receipt(
             interval_seconds=interval_seconds,
         )
 
+    async def __experimental_use_network(
```

Comment:
Looks like double_underscore has a special meaning in Python – this needs to be double_underscored on both sides, so:
```
async def __experimental_use_network__(
```